src/elm/Main.elm

```diff
@@ -684,7 +684,7 @@ header : Model -> String -> Html Msg
 header model contentClass =
     let
         links =
-            [ ( "Ny periodebillett", Route.Shop, Shared.hasPeriodTickets model.shared )
+            [ ( "Ny periodebillett", Route.Shop, Shared.hasNonCarnetTickets model.shared )
```

Comment:
Er det ikke mer riktig at dette kaller "hasPeriodTickets"? Ser at den nå inkluderer singleticket, men enkeltbillett burde jo ha sin egen shop hvor det er mulig å kjøpe flere i samme slengen samt forskjellige brukertyper (f.eks. 2 vokse og 2 barn), altså i en og samme ordre.

Reply:
Jeg gikk litt frem og tilbake her. Per nå er det ikke et ønske for AtB og ha enkeltbilletter, så tenker det kan innføre ekstra kompleksitet på noe som ikke er ønskelig. Dersom andre organisasjoner vil ha det kan det løses med eget språk og bruke distribusjonskanal for å tillate det. Dersom det blir et behov for f.eks NFK med flere kategorier så kan jeg løse det da, men per nå vil vel heller ikke de ha enkeltbilletter. Så da tenkte jeg dette ble mest teknisk korrekt + gir mest fleksibilitet.

Reply:
Jeg er litt usikker, siden det kan være forskjellige behov i UI for forskjellige typer billetter. Synes det hadde vært mest riktig å hatt `Route.ShopSingle`, `Route.ShopPeriod` og `Route.ShopCarnet` med tilhørende tekst og funksjon i `Shared`. Akkurat nå blir "ny periodebillett" (`Route.Shop`) en catch-all for alt, og det tror jeg ikke vi er tjent med.

Vet at AtB ikke ønsker enkeltbillett i webshop, men det burde jo være lett å legge bak en konfigurasjon (enten compile time eller fra backend).

Reply:
Per nå er det ikke et behov for NFK med singles heller, og vi jobber opp mot prod release i november (ja), så kommer ikke til å kunne prioritere et potensielt behov per nå. Men jeg kan fint endre spesifikt til period tickets igjen for med den samme forenklingen som er her, og opprette en anne oppgave for singles.